extmod/vfs_posix_file.c

```diff
@@ -209,6 +209,15 @@ STATIC mp_uint_t vfs_posix_file_ioctl(mp_obj_t o_in, mp_uint_t request, uintptr_
                 if (pfd.revents & POLLOUT) {
                     ret |= MP_STREAM_POLL_WR;
                 }
+                if (pfd.revents & POLLERR) {
+                    ret |= MP_STREAM_POLL_ERR;
+                }
+                if (pfd.revents & POLLHUP) {
+                    ret |= MP_STREAM_POLL_HUP;
+                }
+                if (pfd.revents & POLLNVAL) {
+                    ret |= MP_STREAM_POLL_NVAL;
+                }
```

Comment:
yes, this is me using the latest version, this change was made here
https://github.com/micropython/micropython/commit/22106bf2fa574ba947eceb3b9cfa401c67c6a7eb

Reply:
@stephanelsmith The issue is that you need to rebase this PR on the latest master branch, rather than including the same changes in your commit.

(Your commit's parent is cfcce4b5311c7b112d3628392c8012b4c29bdd63 but you need to rebase after 218242d1de781124307be8720a7a9d2f373ca46b)

Reply:
Thanks!

Reply:
Rebase done, sorry about that.